devtools/conda-envs/rdkit-examples.yaml

```diff
@@ -17,6 +17,7 @@ dependencies:
   - openff-units
   - openff-utilities >=0.1.5
   - openff-interchange-base =0.4
+  - nomkl
```

Comment:
See for context: https://github.com/openforcefield/openff-toolkit/blob/73d62fecd60751d4ee6799e3a23ea3544eab903e/devtools/conda-envs/test_env.yaml#L48-L49

Reply:
Wow, that issue mirrors what I did this time, all the way down to not reproducing the issue in a docker container.